status-server/data/user/0/slots

```diff
@@ -1,11 +1,10 @@
 {
   "value":[
-    "/user/0",
-    "/user/1",
-    "/user/2",
-    "/user/3",
-    "/user/4",
-    "/user/5",
-    "/user/6",
+    "1",
```

Comment:
In my mind these values were URLs, all under the same origin for now, but ultimately living anywhere on the web. I don't actually care much right now as I'm sure this will all see a couple of rewrites between now and then, but just to keep this in mind.